### DISCUSSION TITLE
JMSToolBox multiple instances on same computer

### DISCUSSION THREAD

**Discussion by LeoAhm** [jmstoolbox/jmstoolbox#106] [Q&A]
Hi,

Is it possible to have multiple instances on the same computer.
We have 6-10 users that would like to access JMSToolBox at the same time on a Terminal server for testing options against an IBM MQ server.
Can you just have multiple folders with JMSToolBox on there and share the templates folder?

**Comment by titou10titou10** [comment]
Currently no
